README-en.md

```diff
@@ -0,0 +1,51 @@
+# China Geospatial Data for GMT
+
+![GitHub repo size](https://img.shields.io/github/repo-size/gmt-china/china-geospatial-data)
+
+This project contains China geospatial data that can be used in
+[Generic Mapping Tools (GMT)](https://www.generic-mapping-tools.org/)
+
+-**Homepage**: https://github.com/gmt-china/china-geospatial-data
+-**Download**: https://github.com/gmt-china/china-geospatial-data/releases
+-**Use Instructions**: https://docs.gmt-china.org/latest/dataset-CN/ (In Chinese)
+
+## Data
+
+### China Administrative Divisions
+
+- [CN-border-La.gmt](CN-border-La.gmt): China's national borders, provincial borders,
+  ten-segment lines, and South China Sea island data
+- [CN-border-L1.gmt](CN-border-L1.gmt): China's national borders, ten-segment line,
+  and South China Sea island data, excluding provincial data
+- [ten-dash-line.gmt](ten-dash-line.gmt): only contains ten-segment line data
+
+### China mainland and neighboring blocks
+
+- [CN-block-L1.gmt](CN-block-L1.gmt): Level 1 block boundary data
+- [CN-block-L1-deduced.gmt](CN-block-L1-deduced.gmt): Dedudec Level 1 block boundary data
+- [CN-block-L2.gmt](CN-block-L2.gmt): Level 2 block boundary data
+
+### China Fault
+
+-[CN-faults.gmt](CN-faults.gmt): China faults
+
+### Geological Map of Mainland China and Adjacent Areas
```

Comment:
See the difference between **region** and **area**: http://blog.sina.com.cn/s/blog_540316260100u07h.html
```suggestion
### Geological Map of the Mainland China and Adjacent Regions
```